src/main/java/mate/academy/springboot/datajpa/dto/request/RequestProductDto.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.springboot.datajpa.dto.request;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
let's try not to use Data annotation for entities. It's better to use Getter Setter etc.
please check other entities.